src/app.js

```diff
@@ -728,17 +687,17 @@ function run () {
     app: self.event,
     udapp: udapp.event
   }
-  var righthandPanel = new RighthandPanel(rhpAPI, rhpEvents, {}) // eslint-disable-line
-  self._view.rightpanel.appendChild(righthandPanel.render())
-  righthandPanel.init()
+  self._components.righthandpanel = new RighthandPanel(rhpAPI, rhpEvents, {})
```

Comment:
why keeping `self._components.righthandpanel` ? because it is not used (and `filePanel` which have the same role is not kept in `self._component`) . 
Also in the future, would be nice to rename `righthandpanel` to maybe `tabsPanel` (to more consistent with `filePanel`)